modules/ui/commerce_stock_ui.routing.yml

```diff
@@ -4,12 +4,12 @@ commerce_stock_ui.stock_transactions1:
     _form: '\Drupal\commerce_stock_ui\Form\StockTransactions1'
     _title: 'Create stock transaction'
   requirements:
-    _access: 'TRUE'
+    _permission: 'administer commerce_stock'
```

Comment:
Creating transactions should not be for stock admins especially as this is one of the 3 Entry systems the stock field supports.